docs/operation/common-problems.md

```diff
@@ -96,6 +96,10 @@ If you experience problems while configuring RBAC (Role-Based Access Control) fo
 
 In order to solve the problem, please refer to the [troubleshooting](../installation/teams/index.md#troubleshooting) guide for assistance.
 
+## I can't see my Slack private channels in Cloud Dashboard
+
+In adherence to Slack privacy policy, private channels are not displayed until a public alias is set for the respective channel. Kindly refer to the provided instructions for guidance on completing this process: [Setting Public Alias for Private Channels](../installation/slack/cloud-slack#setting-public-alias-for-private-channels)
```

Comment:
change this to: To comply with Slack's privacy policy, private channels won't be visible until you create a public alias for the respective channel. Refer to the provided instructions for guidance on completing this process: [Setting Public Alias for Private Channels](../installation/slack/cloud-slack#setting-public-alias-for-private-channels)

Reply:
resolved in https://github.com/kubeshop/botkube-docs/pull/321